mix.exs

```diff
@@ -65,7 +69,9 @@ defmodule Ret.Mixfile do
       {:secure_random, "~> 0.5"},
       {:bamboo, "~> 1.3"},
       {:bamboo_smtp, "~> 1.7"},
-      {:guardian, "~> 1.2"},
+      {:guardian, "~> 2.1.1"},
+      {:guardian_phoenix, "~> 2.0"},
+      {:guardian_db, "~> 2.0"},
```

Comment:
Optional dependency to store / revoke / sweep tokens in the database. We could do this ourselves if we prefer -- it's not a big library.